File: chapter5/Cash_in_python/cash.py
from cs50 import get_float
while True:
    # prompts user for proper input
    change = get_float("How much change is owed? \n")
    if(change >= 0):
        break
    
cents = round(change * 100)
coins = 0

quarters = int(cents / 25)
dime = int((cents % 25) / 10)
nickels = int(((cents % 25) % 10) / 5)
pennies = int(((cents % 25) % 10) % 5)

coins = coins + quarters
if dime > 0:
    coins = coins + dime
if nickels > 0:
    coins = coins + nickels
if pennies > 0:
    coins = coins + pennies

print(coins)

File: chapter1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float dollars;
    do
    {
        dollars = get_float("How much change is owed? \n"); // repeat the question until valid input
    }
    while (dollars <= 0); // only alow numbers over 0

    int cents = round(dollars * 100);
    int quarters = cents / 25; // find how many quarters
    int quartersr = cents % 25; // find remainder 
    int dimes = quartersr / 10; // find how many dimes
    int dimesr = quartersr % 10; // find remainder
    int nickles = dimesr / 5; // find how many nickles
    int nicklesr = dimesr % 5; // find remainder
    int pennies = nicklesr; // make the pennies variable equal to the remainder.
    int total = quarters + dimes + nickles + pennies;
    printf("%i quarters, %i dimes, %i nickles, and %i pennies", quarters, dimes, nickles, pennies);
    printf("%i /n", total);


}